csharp/ql/lib/semmle/code/csharp/dataflow/internal/ExternalFlow.qll

```diff
@@ -310,10 +328,17 @@ class UnboundCallable extends Callable {
   }
 }
 
+private predicate hasName(Declaration d, string name) {
+  d.(Operator).getFunctionName() = name
+  or
+  not d instanceof Operator and
+  d.hasName(name)
+}
+
 pragma[nomagic]
 private predicate callableSpecInfo(Callable c, string namespace, string type, string name) {
   c.getDeclaringType().hasQualifiedName(namespace, type) and
-  c.getName() = name
+  hasName(c, name)
```

Comment:
🙀 So was it the case, that even though the models existed they were not applied correctly to the operator?

Reply:
Yes.